opm/simulators/flow/Main.hpp

```diff
@@ -90,6 +89,8 @@ struct FlowEarlyBird {
 
 namespace Opm {
 
+void printFlowBanner(int nprocs, int nthreads);
```

Comment:
Yeah, I hoped to avoid including the header in simulator objects, but it turned out it was not doable, so this forward is not worth it. Will change to include.

Reply:
done.